core/src/components/checkbox/checkbox.tsx

```diff
@@ -233,6 +255,7 @@ export class Checkbox implements ComponentInterface {
             <svg class="checkbox-icon" viewBox="0 0 24 24" part="container">
               {path}
             </svg>
+            {theme === 'ionic' && <div part="focus-ring" class="focus-ring"></div>}
```

Comment:
Be sure to document this shadow part at the top of the file. You can checkout what was done for button on the latest `next` branch. We should make it clear this shadow part is only available in the ionic theme.